### PR TITLE
Add content type filtering to the resources list

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added a select element to filter the content type. Added the functionality to filter by type, topic and both. Moved the navigation item to the left of downloads.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/resources](http://0.0.0.0:8001/resources)
- Filter some and check it works

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/264

### CODE REVIEW

**Comment by anthonydillon** [comment]
@frankiegranato has seen this branch and :ok: it.

File: webapp/views.py [hunk 1]
```diff
@@ -69,20 +74,55 @@ def _normalise_resources(self, resources):
...
-            )
-            feed_items[topic] = {}
-            feed_items[topic]['items'] = get_json_feed_content(api_url)
-            feed_items[topic]['group_name'] = self.GROUPS[topic]['name']
+        if topic or content:
+            if not content:
+                category_id = str(self.GROUPS[topic]['id'])
+                api_url = (
+                    '{api_url}/posts?_embed'
+                    '&group={category_id}&{resource_filter}'
```
Comment (grahambancroft) [inline webapp/views.py]: Do the ampersands in the urls need to be escaped?

**Comment by grahambancroft** [comment]
Couple of issues on small, arrow on right hides covers links and the content dropdown is difficult to get to
 
screen shot 2017-11-28 at 10 29 47

**Comment by grahambancroft** [comment]
Code 👍

File: webapp/views.py (hunk 1 above)
Comment (anthonydillon) [reply webapp/views.py]: No, they are a string that represent the URI to the data endpoint.

**Comment by anthonydillon** [comment]
Thanks @grahambancroft updated the filtering section to work on all viewports.

**Comment by grahambancroft** [comment]
I'm still seeing the same issue with the chevron on the right covering the links

**Pull request merged**